### PR TITLE
Add Japanese translation for base navigation and home page

### PR DESCRIPTION
**Pull request by shuuji3**
## Related Issue

N/A

## Summary of Changes
I have translated the base navigation text and home page text into Japanese, using #150 as a reference.

### CODE REVIEW

**Review by sbesh91 — COMMENTED** [review]
These are looking structured correctly to me.  I unfortunately do not speak or read Japanese, so I can't review the content of these translations.  We currently use a scheme where we auto-redirect based on the default language set for the browser.  A new dropdown widget to link to other translations would be a great addition in another pr.

File: src/ja/home.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+<p class="home-intro">
+  私たちは<strong>オープンなウェブの未来</strong>を擁護するために世界中から集まったソフトウェアエンジニアのグループです。規制当局、立法者、政策立案者に対して複雑な技術の詳細情報を提供することで、この業界における競争を阻害する主要な問題とその解決方法の理解をサポートしています。
+</p>
+
+<div class="highlight">
+  <h2>危機にさらされているのは何か？</h2>
+
+  <div class="group">
+    <p>アプリケーション開発の未来すべてが危機にさらされています。規制や立法による変化なくしては、普遍的で、無料かつオープンな、一度書くだけでよく、どこにでもデプロイ可能な、アプリケーション頒布とデプロイのシステムを失うリスクがあります。このシステムは、企業と消費者のコストを大幅に削減するシステムです。
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:17]:
```suggestion
    <p>アプリケーション開発の未来すべてが危機に瀕しています。規制や法律の変化が起こらなければ、企業と消費者の双方の負担を大幅に削減するアプリケーション公開の場が失われるおそれがあります。一度作ればどこにでも配布・導入できる、普遍的で、無料で、開かれたシステムが失われる可能性があるのです。
```
一文がすごく長くなってしまうのは避けたいが……

File: src/ja/home.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    <p>もし規制や立法による変化がなければ、資金はプロプライエタリなエコシステムに向かい、ゲートキーバーたちは重税を徴収することが可能になります。これにより、ゲートキーバーのコントロール下にロックインされ、今後何年にもわたってモバイルアプリのイノベーションが阻害されることになってしまいます。</p>
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:18]: - 文脈的に、"funding"が指してるのは"資金"ではなく"企業"？"ビックテック"に変えてもいいかも
- "ゲートキーパー"の説明を多少入れたほうがいいかも……？
```suggestion
    <p>もし規制や法律の変化がなければ、大企業は独占的な生態系に移行し、「ゲートキーパー」と呼ばれる企業たちは重い"税金"を徴収できるようになります。これにより、ゲートキーバー達の管理化に囚われ、今後長きにわたってモバイルアプリのイノベーションが阻害されることになってしまいます。</p>
```

File: src/ja/home.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  </div>
+</div>
+
+<div class="presentation">
+  <div class="policy-entry">
+    <h2>どこから始めればいい？</h2>
+    <p>開発者のアドホック・グループとして、私たちは<a href="/ja/apple-browser-ban">#AppleBrowserBan</a>（Appleによるブラウザーの禁止）がオープンなウェブの未来に対する大きな脅威であると確信しました。イギリスの独占規制当局である、競争・市場庁（Competition and Markets Authority、CMA）が<a href="https://www.gov.uk/cma-cases/mobile-ecosystems-market-study">モバイルエコシステムに対する市場調査</a>を立ち上げた際に、何人かのイギリスの「メンバー」が、CMAに対してブリーフィングを行うために招待されました。</p>
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:25]:
```suggestion
    <p>開発者の特別な集まりとして、私たちは<a href="/ja/apple-browser-ban">#AppleBrowserBan</a>（Appleによるブラウザーの禁止）が開かれたウェブの未来に対する大きな脅威であると確信しました。イギリスの独占規制当局である、競争・市場庁（Competition and Markets Authority、CMA）が<a href="https://www.gov.uk/cma-cases/mobile-ecosystems-market-study">モバイルエコシステムに対する市場調査</a>を立ち上げた際に、何人かのイギリスの「メンバー」が、CMAに対して報告を行うために招待されました。</p>
```

File: src/ja/home.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+    <p>私たちはこの調査の一環として、規制当局への提出書『<a href="/ja/walled-gardens-report/">Bringing Competition to Walled Gardens（ウォールド・ガーデンに競争をもたらす）</a>』を作成しました。</p>
+
+    <!-- <p><a href="/ja/policy/" class="action-button">About our actions</a></p> -->
+  </div>
+
+  <div class="priorities-entry">
+    <h2>優先すべきことトップ3</h2>
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:33]:
```suggestion
    <h2>3つの最優先事項</h2>
```

File: src/ja/home.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+    <dl class="counter">
+      <dt>
+        <a href="/ja/apple-browser-ban">#AppleBrowserBan</a>
+      </dt>
+      <dd><strong>Appleによるサードパーティ製ブラウザの禁止</strong>は、iOS上で行われてきた非常に反競争的な行為です。Safari/WebKitチームの資金援助を制限することで、過去10年間に渡ってイノベーションを停滞させ、モバイル上でウェブアプリが発展することを妨げてきました。</dd>
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:39]: - "資金援助を制限する" ← これ、制限しているのは"資金"ですか？もし人による仕事を含めた"資源"なのであれば表現を変えるべき
- 過去形ですか？AppleBrowserBanが今も残っているなら語尾を現在進行形に変えたほうがいい
```suggestion
      <dd><strong>Appleによるサードパーティ製ブラウザの禁止</strong>は極めて反競争的な行為です。これはSafari/WebKitチームの活動を制限することで、過去10年間に渡ってイノベーションを停滞させ、モバイル上でウェブアプリが発展することを妨げています。</dd>
```

File: src/ja/home.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+      <dt>深いシステム統合</dt>
+      <dd><strong>ウェブアプリは普通のアプリになる必要があります。</strong>無料でオープンなウェブ技術で構築されたアプリには、平等な扱いと統合が必要です。閉鎖的で重税が課されたプロプライエタリなエコシステムは、いかなる優遇も受けるべきではありません。
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:42]:
```suggestion
      <dd><strong>ウェブアプリは普通のアプリになるべきです。</strong>無料で開かれたウェブ技術で構築されたアプリには、平等な扱いと統合が必要です。閉鎖的で開発者に重い負担がかかる独自のエコシステムは、いかなる優遇も受けるべきではありません。
```

File: src/ja/home.html [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+      <dt>ウェブアプリの平等性</dt>
+      <dd><strong>ゲートキーバーたちによって設置された人工的な障壁は、すべて取り除かなければなりません。</strong>ウェブアプリは、許可さえあれば、要求の厳しいユースケースに対して、より優れたプライバシーとセキュリティを備えた同等の機能を提供できます。</dd>
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:45]: - "要求の厳しい"はちょっとこっちが悪者感でてるかも。"より多くの"とかに変えたほうがいいと思う
- "ユースケース"は非カタカナ語にするのはむずかしそう
- "プライバシー"・"セキュリティ"は一般語彙になっていると思う
- "can"は[これ](https://eow.alc.co.jp/search?q=can)の7個目の意味で使ってませんか？もしそうなら最後を以下の通り変えたほうがいい
```suggestion
      <dd><strong>ゲートキーバーたちによって設置された人工的な障壁は、すべからく取り除かなければなりません。</strong>許可されたウェブアプリは、より多くのユースケースに対して、より優れたプライバシーとセキュリティを備えて同等の機能を提供できるべきです。</dd>
```

File: src/ja/home.html [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+      <h2>ぜひ支援してください</h2>
+      <p>イギリス、EU、日本、オーストラリア、アメリカ合衆国の規制当局は、Appleのサードパーティ製ブラウザエンジンの禁止に対して捜査を開始しており、ウェブアプリがネイティブアプリと競合できるようにするための介入を検討しています。最も重要な点は、この趣旨の文言がEUの画期的な技術法であるデジタル市場法にすでに含まれていることです。</p>
+
+      <p>これは必然の結果ではありません。過去2年間に渡って、Open Web Advocacyが規制当局や議員に対して、モバイルアプリの複占の影響を相殺できるウェブの可能性について、広範囲に説明してきた結果です。</p>
+
+      <p>私たちの擁護活動のほぼ確実な結果として、AppleはWebKitとSafariの開発に携わる多数の追加スタッフを雇用しました。この追加のリソースと規制の圧力は、ウェブアプリのプッシュ通知やアプリのバッジなどの長期間延期されていた機能の追加に直接貢献しました。WebKitのスタッフ数は秘密にされていますが、これは大幅な増加だと考えられます。</p>
+
+      <p><strong>競争、あるいは少なくともその脅威は、正しく機能するのです。</strong></p>
+
+      <p>競争を取り戻し、ユニバーサルなウェブアプリを現実のものとするために圧力をかけ続けるために、OWAにはあなたの助けが必要です。この取り組みをサポートすれば、ウェブの将来を確実に変えることができます。ウェブの発展を阻止するためにあらゆるインセンティブを持つ、信じられないほど潤沢な資金を持つゲートキーバーたちに挑戦するために、私たちは寄付に頼っています。</p>
```
Comment (AXT-AyaKoto) [inline src/ja/home.html:62]: - "インセンティブ"はビジネスシーンでしか使われない印象。別の表現に変えたほうがいいかな……
```suggestion
      <p>競争を取り戻し、ユニバーサルなウェブアプリを現実のものとするための圧力をかけ続けるために、OWAにはあなたの助けが必要です。この取り組みをサポートすれば、ウェブの将来を確実に変えることができます。ウェブの発展を阻止するためにあらゆる行動を起こす、信じられないほど潤沢な資金を持つゲートキーバーたちに挑戦するために、私たちは寄付に頼っています。</p>
```

**Review by AXT-AyaKoto — COMMENTED** [review]
For now, I'll read through it briefly and send you some suggested changes for the parts that caught my eye.

**Comment by jessbudd** [comment]
So exciting to see another language being added to the site! Thank you @shuuji3 for all your effort in translating and @AXT-AyaKoto for reviewing 🫶 

I notice there's some merge conflicts now, I assume due to the [language picker](https://github.com/OpenWebAdvocacy/website/pull/173) being added recently.  Are we able to keep moving this PR forward? Looks so close!

**Comment by shuuji3** [comment]
@AXT-AyaKoto 確認が大変遅くなってしまってすみません。レビューありがとうございました。より自然な翻訳にできたと思います。

**Comment by shuuji3** [comment]
@jessbudd OK, I've resolved the conflict and added a few new translations, including a small typo fix for the navigation text in English.

screenshot of website top page in Japanese

Screenshot from 2024-11-30 23-57-51

File: src/_data/languages_base.json [hunk 9]
```diff
@@ -1,12 +1,13 @@
...
-    {"code": "es", "base": "/es", "name": "Español"}
+    {"code": "es", "base": "/es", "name": "Español"},
+    {"code": "ja", "base": "/ja", "name": "日本語"}
   ],
   "en": {
     "ariaLabelPaginationlinks": "Pagination links",
-    "ariaLabelPrimaryNav": "Primary navigaton",
-    "ariaLabelSecondaryNav": "Secondary navigaton",
+    "ariaLabelPrimaryNav": "Primary navigation",
+    "ariaLabelSecondaryNav": "Secondary navigation",
```
Comment (jessbudd) [inline src/_data/languages_base.json:10]: Good catch 👍

**Comment by jessbudd** [comment]
Awesome!

@AXT-AyaKoto did you have any further comments on the translations before @sbesh91 can merge this in?

**Pull request merged**